pandas/core/indexes/multi.py

```diff
@@ -176,6 +176,15 @@ def new_meth(self_or_cls, *args, **kwargs):
     return new_meth
 
 
+def _lexsort_depth(codes: List[np.ndarray], nlevels: int) -> int:
```

Comment:
can you move this to almost the end of the file, e.g. where the module level functions are